core/src/test/java/com/linecorp/armeria/server/ServiceBindingTest.java

```diff
@@ -126,6 +158,7 @@ void consumesAndProduces() throws IOException {
                                            .aggregate().join();
         assertThat(res.status()).isSameAs(HttpStatus.OK);
         assertThat(res.contentUtf8()).isEqualTo("armeria");
+        assertThat(decoratorLog.stream().collect(Collectors.toList())).isEqualTo(Arrays.asList(3, 2, 1));
```

Comment:
`isEqualTo` could be simplified with `containsExactly(3, 2, 1)`

Reply:
Thanks. Done.